### DISCUSSION TITLE
Issues tabs has been temporarily hidden

### DISCUSSION THREAD

**Discussion by victornpb** [victornpb/undiscord#276] [# Announcements]
This repository has currently ~100 opened issues. A lot of them are questions or not-a-bug, but reading them all and making sure it's not a bug takes time. I'm disabling the issues tab temporarily until I can go through all of them.

In the meantime, I want to experiment using the _Discussions_ tab. This should make helping other members easier.

I will turn the _issues_ tab back on eventually, but I want to only have verified bugs there because it will make fixing them much easier (for me and for contributors).

**Comment by victornpb** [comment]
Thank you for all the contributors that have been replying issues there ❤️

**Comment by pinkforest** [comment]
@victornpb victornpb/deleteDiscordMessages/issues/193 that is linked from README.md now goes 404

re: discussion about self-bots - perhaps moving that to under discussions?

**Reply by elaine-jackson** [reply]
I was wondering about this too

**Reply by victornpb** [reply]
done! thank you
